iosApp/iosApp/ContentView.swift

```diff
@@ -118,10 +172,11 @@ struct ContentView: View {
             } else {
                 ZStack(alignment: .top) {
                     mapWithSheets
-                    VStack(alignment: .trailing, spacing: 0) {
+                    VStack(alignment: .center, spacing: 0) {
                         if nearbyVM.navigationStack.lastSafe() == .nearby {
                             SearchOverlay(searchObserver: searchObserver, nearbyVM: nearbyVM, searchVM: searchVM)
                         }
+                        locationAuthButton
```

Comment:
The button is unexpectedly still showing when search is opened. Similar to the RecenterButton below, I think a `if !searchObserver.isSearching` check to remove this would do the trick

![IMG_0166](https://github.com/user-attachments/assets/2ab7608c-0d9a-456a-a21a-a546e3343854)

Reply:
Also seeing some unexpected alignment in the stop details page - button is no longer centered

![IMG_0167](https://github.com/user-attachments/assets/1969a840-ddcf-401f-9fde-013531d3fc9c)

Reply:
Oh odd, thought I tested that early on. Maybe I messed something up in the rebase, I'll check

Reply:
Ah, it was my fault for not waiting long enough! The button appears to be hidden by the keyboard at first, but then shifts to be visible on the screen if you wait a second